lambda_src/drivers/process_google.py

```diff
@@ -1,7 +1,7 @@
 from json import loads
 
-from google.oauth2 import credentials, service_account  # type: ignore
-from googleapiclient.discovery import build  # type: ignore
```

Comment:
why remove ignoring of type? is there a new type definition somewhere?

Reply:
No. Sorry. Was trying to see if latest version of the stubs has typing. It still doesn't.

Reply:
then can you undo these changes?

Reply:
The discovery stubs are available and we don't need to ignore it. for google.oauth2 stubs are not available and hence we still need to ignore.